visualization/src/resource-visualization.cc

```diff
@@ -63,6 +64,16 @@ DEFINE_bool(
     vis_pointcloud_color_random, false,
     "If enabled, every point cloud receives a random color.");
 
+DEFINE_bool(
+    vis_pointcloud_filter_dense_map_before_publishing, true,
+    "If enabled, the accumulated dense map will be filtered before "
```

Comment:
Also works for the non-accumulated one...
```suggestion
    "If enabled, the visualized dense map will be filtered before "
```